examples/https_attestation/backend/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "https_attestation_backend"
+version = "0.1.0"
+authors = ["Ivan Petrov <ivanpetrov@google.com>"]
+edition = "2018"
+license = "Apache-2.0"
+
+[dependencies]
+anyhow = "*"
+http = "*"
+hyper = "*"
+env_logger = "*"
+futures = "*"
+futures-core = "*"
+futures-util = "*"
+log = "*"
+structopt = "*"
+# Pinned to 0.2 because of tonic: https://github.com/hyperium/tonic/blob/master/tonic/Cargo.toml
```

Comment:
This should no longer be needed, and might even cause some issues, after the upgrade of Tonic.

Reply:
Done